rust/lance/src/dataset/scanner.rs

```diff
@@ -1537,7 +1530,7 @@ impl Scanner {
         let schema = fts_node.schema();
         let group_expr = vec![(expressions::col(ROW_ID, &schema)?, ROW_ID.to_string())];
         let fts_node = Arc::new(AggregateExec::try_new(
-            AggregateMode::Final,
+            AggregateMode::Single,
```

Comment:
Why this change?

Reply:
`Final` is used for combining the partial results